docs/contributing/how-to-update-the-build-image.md

```diff
@@ -10,6 +10,5 @@ The build image currently can only be updated by a Cortex maintainer. If you're
 1. Update `build-image/Docker`
 2. Build the image running `make build-image/.uptodate`
 3. Publish the image to the repository running `docker push quay.io/cortexproject/build-image:TAG` (this can only be done by a maintainer)
-4. Replace the image tag in `.circleci/config.yml` (_there may be multiple references_)
-5. Replace the image tag in `.github/workflows/*` (_there may be multiple references_)
-6. Open a PR and make sure the CI with new build-image passes
+4. Replace the image tag in `.github/workflows/*` (_there may be multiple references_)
```

Comment:
Are there?  Can we fix that?

Reply:
There are and, to my knowledge, they're not easy to fix because can't have default values in GitHub actions yaml definition. I'm not aware of an easy way to fix it.